ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.less

```diff
@@ -2,5 +2,7 @@
   > .collection {
     margin: auto;
     max-width: 1020px;
+    height: 100%;
+    overflow: auto;
```

Comment:
It might be worth moving these two css properties one dom element higher so you aren't imposing  `max-width` on the scrollbar